### PR TITLE
PBF support for `/expansion`

### PR DESCRIPTION
**Pull request by chrstnbwnkl**
## Tasklist

 - [x] Add tests
 - [ ] Add #fixes with the issue number that this PR addresses
 - [ ] Update the docs with any new request parameters or changes to behavior described
 - [x] Update the [changelog](CHANGELOG.md)
 - [ ] If you made changes to the lua files, update the [taginfo](taginfo.json) too.

### CODE REVIEW

**Review by nilsnolde — CHANGES_REQUESTED** [review]
I have absolutely nothing to add here, good job:) 

Well, other than extending the expansion test to prove the PBF stuff.

**Comment by nilsnolde** [comment]
also don't feel like you'd have to wait for https://github.com/valhalla/valhalla/issues/4601. it's a bit unfortunate, but not a big deal to do it after this PR was merged to master.

**Comment by chrstnbwnkl** [comment]
I thought there was no necessity for new tests since the protobuf representation is used anyway for the default JSON serialization.

**Comment by nilsnolde** [comment]
Right, no detailed tests, but there should somewhere be a pbf output test for expansion IMO.

File: src/thor/expansion_action.cc [hunk 1]
```diff
@@ -6,9 +6,53 @@
...
 
 using namespace rapidjson;
 using namespace valhalla::midgard;
+using namespace valhalla::tyr;
+
+namespace {
+
+using namespace valhalla;
+
+void writeExpansionProgress(Expansion* expansion,
```
Comment (chrstnbwnkl) [inline src/thor/expansion_action.cc:19]: snake_case this

**Review by nilsnolde — APPROVED** [review]

**Pull request merged**